File: 백준/Bronze/10178. 할로윈의 사탕/할로윈의 사탕.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();

        for (int i = 0; i < n; i++) {
            int totalPieces = sc.nextInt();
            int people = sc.nextInt();

            int quotient = totalPieces / people;
            int remainder = totalPieces % people;

            System.out.println("You get " + quotient + " piece(s) and your dad gets " + remainder + " piece(s).");
        }
    }
}

File: 백준/Bronze/10178. 할로윈의 사탕/할로윈의 사탕.cc
#include <iostream>
using namespace std;

int main() {

	int n;
	cin >> n;

	for (int i = 0; i < n; i++)
	{
		int totalPieces;
		int people;

		cin >> totalPieces >> people;

		int getPieces = totalPieces / people;
		int rest = totalPieces % people;

		cout << "You get " << getPieces << " piece(s) and your dad gets " << rest << " piece(s)." << endl;
	}
}